gui/gui.go

```diff
@@ -84,6 +84,30 @@ func (ui *GUI) route() {
 	ui.router.HandleFunc("/admin", ui.PostAdmin).Methods("POST")
 	ui.router.HandleFunc("/backup", ui.PostBackup).Methods("POST")
 	ui.router.HandleFunc("/logout", ui.PostLogout).Methods("POST")
+
+	// Websocket endpoint allows the GUI to receive updated values
+	ui.router.HandleFunc("/ws", ui.RegisterWebSocket).Methods("GET")
+
+	socketUpdate := func() {
```

Comment:
`socketUpdate` isn't being reused, doesn't need to be a anonymous function as a result. Let's fetch the stats and call `SendUpdatedValues` in the goroutine on a tick instead.